packages/js/email-editor/src/blocks/powered-by-mailpoet/block.tsx

```diff
@@ -4,7 +4,7 @@ import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
 import { PanelBody, RadioControl, Icon } from '@wordpress/components';
```

Comment:
@triple0t Sorry for the late comment! I thought I posted it on Wednesday, but I had it as a pending comment 🤦

Reply:
@costasovo Thank you for the suggestion. That would indeed simplify the folder structure.

I have a question about the `powered-by-mailpoet` block. It is currently using data from the [email editor](https://github.com/mailpoet/mailpoet/blob/9be3d76a2c31f9a18b790689eb14a53f362e955a/packages/js/email-editor/src/blocks/powered-by-mailpoet/block.tsx#L35-L40), how would you suggest we handle this?

Reply:
> I have a question about the powered-by-mailpoet block. It is currently using data from the [email editor](https://github.com/mailpoet/mailpoet/blob/9be3d76a2c31f9a18b790689eb14a53f362e955a/packages/js/email-editor/src/blocks/powered-by-mailpoet/block.tsx#L35-L40), how would you suggest we handle this?

@triple0t I see the CDN url and isPremiumPlugin active. I think we need to put those out of the store because both are related only to MailPoet, so either we could create a new store only for MailPoet stuff, or, as these two don't change dynamically, we could read them from a global space for now.

Reply:
True. Noted. I will make the update.
Thanks @costasovo